lib/passwordless/token_generator.rb

```diff
@@ -0,0 +1,23 @@
+require 'openssl'
+
+module Passwordless
+  class TokenGenerator
+    def initialize(key_generator, friendly_token_generator, digest)
+      @key = key_generator.generate_key('passwordless')
```

Comment:
Devise uses the [column name](https://github.com/heartcombo/devise/blob/master/lib/devise/token_generator.rb#L29) but I've just called it `'passwordless'`